README.md

```diff
@@ -1,24 +1,26 @@
-# Block Template
+attribute_selector
+==================
 
-This repository serves as a "starter" repository for creating a new block.
+This block is used to whitelist or blacklist incoming signal attributes and notify 
+the resulting modified signals.
 
-## How to use
+Properties
+--------------
+-  specify_behavior: specify whitelist or blacklist behavior
```

Comment:
These bullet points should be the labels on the builder, not the names of the variables. Not every block does this but this is a standard I want to move towards. If people are building services and need to know what a block does, they shouldn't have to look at the code to find out what each property does.